webcompat/static/js/lib/flash-message.js

```diff
@@ -32,20 +32,22 @@ var FlashMessageView = Backbone.View.extend({
     var timeout = opts.timeout || 4000;
     var message = opts.message;
 
+    this.$el.addClass("notification-information");
     this.render(message);
     setTimeout(_.bind(this.hide, this), timeout);
   },
   showError: function(data) {
-    this.$el.addClass("is-error");
+    this.$el.addClass("is-error notification-alert");
     this.show(data);
   },
   showForever: function(opts) {
     var message = opts.message;
+    this.$el.addClass("notification-information");
     this.render(message);
   },
   showThanks: function(opts) {
     var buildTemplate = wcTmpl["issue/thanks.jst"];
-    this.$el.addClass("is-active wc-FlashMessage--thanks");
+    this.$el.addClass("is-active notification-information");
```

Comment:
This will get a double `notification-information` class, because `show` adds one too.